src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -422,15 +444,33 @@ public void testVisitTokenSwitchReflection() {
         final DetailAST astSemi = mockAST(TokenTypes.SEMI, ";", "mockfile", 0, 0);
         astIdent.addNextSibling(astSemi);
 
-        // Set unsupported option
-        final ImportOrderCheck mock = new ImportOrderCheck();
-        final ImportOrderOption importOrderOptionMock = PowerMockito.mock(ImportOrderOption.class);
-        Whitebox.setInternalState(importOrderOptionMock, "name", "NEW_OPTION_FOR_UT");
-        Whitebox.setInternalState(importOrderOptionMock, "ordinal", 5);
-        Whitebox.setInternalState(mock, "option", importOrderOptionMock);
-
-        // expecting IllegalStateException
-        mock.visitToken(astImport);
+        final Locale defaultLocale = Locale.getDefault();
+
+        try {
+            // fix for turkey locale which changes the text importOrderOption in
+            // the exception
+            Locale.setDefault(Locale.ENGLISH);
```

Comment:
I do not like this fix .... please put a CS suppression on this method and restore it.
I what to review this case in separate PR, and let all other changes by applied to master.